scripts/minikube.sh

```diff
@@ -203,6 +212,14 @@ up)
 
     disable_storage_addons
 
+    #  get kubernetes version we are operating on and accordingly enable feature gates
+    KUBE_MAJOR=$(kube_version 1)
+    KUBE_MINOR=$(kube_version 2)
+    if  [ "${KUBE_MAJOR}" -eq 1 ] && [ "${KUBE_MINOR}" -ge 22 ];
+    then
+    # if kubernetes version is greater than 1.22 enable RWOP feature gate
+    K8S_FEATURE_GATES="${K8S_FEATURE_GATES},ReadWriteOncePod=true"
```

Comment:
indention is off?

Reply:
done.

Reply:
no, I don't thinks so?

Reply:
@nixpanic looks like I missed to commit, done now.. Thanks!